ros2controlcli/README.md

```diff
@@ -3,12 +3,32 @@ Command line interface for controller manager
 
 ## Verbs 
 
+### configure_controller
 
-### list
+```bash
+$ ros2 control configure_controller -h
+usage: ros2 control configure_controller [-h] [--spin-time SPIN_TIME] [-c CONTROLLER_MANAGER] [--include-hidden-nodes] controller_name
+
+Configure a controller in a controller manager
+
+positional arguments:
+  controller_name       Name of the controller
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --spin-time SPIN_TIME
+                        Spin time in seconds to wait for discovery (only applies when not using an already running daemon)
+  -c CONTROLLER_MANAGER, --controller-manager CONTROLLER_MANAGER
+                        Name of the controller manager ROS node
+  --include-hidden-nodes
+                        Consider hidden nodes as well
+```
+
```

Comment:
Isn't this deprecated. If so please remove.

Reply:
+1 since we're breaking command API anyway, might as well remove the deprecated ones.